iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/extensions/ActivePowerControlImpl.java

```diff
@@ -116,24 +170,26 @@ public void allocateVariantArrayElement(int[] indexes, int sourceIndex) {
     }
 
     @Override
-    public OptionalDouble getMinPOverride() {
-        double result = minPOverride.get(getVariantIndex());
-        return Double.isNaN(result) ? OptionalDouble.empty() : OptionalDouble.of(result);
+    public OptionalDouble getMinTargetP() {
+        double result = minTargetP.get(getVariantIndex());
+        return Double.isNaN(result) ? OptionalDouble.empty() : OptionalDouble.of(withinPMinMax(result, getExtendable()));
     }
 
     @Override
-    public void setMinPOverride(double minPOverride) {
-        this.minPOverride.set(getVariantIndex(), minPOverride);
+    public void setMinTargetP(double minTargetP) {
+        checkTargetPLimits(minTargetP, maxTargetP.get(getVariantIndex()));
+        this.minTargetP.set(getVariantIndex(), checkTargetPLimit(minTargetP, "minTargetP", getExtendable()));
```

Comment:
The similarity of the method names makes the code a little confusing.
Can you rename `checkTargetPLimits` (to `checkLimitsOrder` for instance) to make it easier to read?

Reply:
Done